docs/shared/nx-core.md

```diff
@@ -172,27 +215,24 @@ that adding a `require()` creates a dependency and that some dependencies cannot
 
 Running `yarn nx run-many --target=test --all` will test all projects in parallel.
 
-Running `yarn nx run-many --target=build --projects=app1,app2` will build `proj1` and `proj2` and their
-dependencies in parallel. Note that if `app1` depends on the output of its dependency (e.g., `shared-components`), Nx
-will build `shared-components` first and only then will build the app.
+// placeholder (should we have people make these projects so they can see more in the dep-graph and commands won't fail?)
+Running `yarn nx run-many --target=build --projects=app1,app2` builds `proj1` and `proj2` and their
+dependencies in parallel. Note that if `app1` depends on the output of its dependency (e.g., a `shared-components` library), Nx
+will build the `shared-components` library first and only then build the app.
```

Comment:
it would be good to have people create these apps/projects so they can see more in the dep-graph or mention that these are other things nx core can do otherwise people might be wondering why the commands will fail.